pkg/barbicanapi/deployment.go

```diff
@@ -107,7 +107,7 @@ func Deployment(
 	// Add PKCS11 volumes
 	if slices.Contains(instance.Spec.EnabledSecretStores, "pkcs11") {
```

Comment:
like mentioned above could change `pkcs11` => `barbicanv1beta1.SecretStorePKCS11` when introduce the consts